### PR TITLE
chore(format): enable formatting for /howto, and format /howto.

### PR DESCRIPTION
**Pull request by pepopowitz**
Partially addresses #883. 

Enables prettier formatting for the ./howto folder, and formats the files in that folder.

### CODE REVIEW

File: howtos/technical-writing-cheatsheet.md [hunk 1]
```diff
@@ -10,42 +10,41 @@ Our primary goal in documentation is to achieve organization, clarity, and direc
...
-| Numbers | Write whole numbers one through nine in full. Write whole numbers 10 and upwards as numerals. | In this example, we will create 1 diagram. | In this example, we will create one diagram. |
-| Spelling | Default to American spelling and US English. Visit the [Oxford American Dictionary](https://www.oxfordreference.com/view/10.1093/acref/9780195392883.001.0001/acref-9780195392883) for details. | Analyse<br><br>Bernd Rücker | Analyze<br><br>Bernd Ruecker |
-| Voice/Tense | Second person, [active voice](https://www.grammarly.com/blog/active-vs-passive-voice/#:~:text=Active%20voice%20means%20that%20a,it%20isn't%20that%20simple.). | I, me, my.<br><br>The computer is turned on by pressing the power button. | You, your.<br><br>Press the power button to turn on the computer. |
+| Subject     | Practice                                                                                                                                                                                        | Avoid                                                                     | Example/Use                                                       |
```
Comment (pepopowitz) [inline howtos/technical-writing-cheatsheet.md:13]: I could see the formatting of these tables being controversial. They look much nicer in an editor, with word-wrap turned off: 

image

If we collectively don't like this format, we can revert to the compact form.

There is no setting for "compact tables" -- [prettier doesn't want to have an opinion on that](https://github.com/prettier/prettier/issues/5651) -- but I _can_ make prettier use the compact version for these tables by setting [`proseWrap` to `never`](https://prettier.io/docs/en/options.html#prose-wrap). That does have the side effect of never wrapping long paragraphs in Markdown, but it looks like we don't wrap paragraphs intentionally anyway.

Comment (christinaausley) [reply howtos/technical-writing-cheatsheet.md:13]: From a technical writing perspective, this looks great to me.

**Review by christinaausley — DISMISSED** [review]
This cleanup makes my technical writing heart so happy. Approving from that perspective! Interested to hear @korthout and @menski thoughts on the table wrapping.

**Comment by pepopowitz** [comment]
I have to resolve conflicts on this one anyway, so I'm going to wait until early next week to merge this one (and hope that I get another review by then)

File: howtos/technical-writing-cheatsheet.md (hunk 1 above)
Comment (pepopowitz) [reply howtos/technical-writing-cheatsheet.md:13]: @christinaausley I'm done waiting for reviews on this one -- mind giving me another green review so I can merge, and if anyone comes along expressing dislike for the table formatting, we can revisit?

**Pull request merged**